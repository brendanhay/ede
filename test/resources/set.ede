{
  "var": true
}
---
{% set x %}

  {% set y %}
    This is a nested block {{var}}
  {% endset %}

  This is a big expression! {{var}}

  {{ y }}

{% endset %}

{{x}}