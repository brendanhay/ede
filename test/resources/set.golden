


  This is a big expression! true

      This is a nested block true


